boards/arm/mimxrt1064_evk/lpm.c

```diff
@@ -0,0 +1,312 @@
+/*
```

Comment:
Is there a reason this file cannot be in the `soc` folder?

Reply:
No there is not. I have moved it to the `soc` folder in the latest push and made the clock code read all PLL configurations at boot, so that changes to PLL dividers on custom boards can be accounted for.